modules/statistics/templates/table_statistics.tpl

```diff
@@ -44,199 +44,203 @@
 </div>
 
 <br>
-<table id="bigtable" class="data table table-primary table-bordered">
-    <tr>
-        <th rowspan="1" id="tpcol">Subproject</th>
-        {assign var='colspan' value=count($Subcategories)}
-        {foreach key=proj item=name from=$Subprojects}
-            <th colspan="{$colspan}">{$name|capitalize}</th>
-        {/foreach}
-        <th colspan="{$colspan}">Total Across Subprojects</th>
-    </tr>
-    <tr>
-        <th>Categories</th>
-        {foreach key=proj item=name from=$Subprojects}
-            {* Go through each category once, and add the total
-               for each cohort *}
+<table id="bigtable" class="data table table-primary table-bordered dynamictable">
+    <thead>
```

Comment:
note: adding "dynamictable" on line 47 and <thead> and <tbody> are the only non-trivial changes here. everything else is just a result of properly indenting elements within <thead> and <tbody>

Reply:
Adding `?w=1` to the URL confirms this..

Reply:
^ this works 😮 . append `?w=1` to the url of the 'files changed' page to remove whitespace changes when reviewing